pkg/dockerregistry/server/auth.go

```diff
@@ -345,18 +359,15 @@ func verifyImageStreamAccess(ctx context.Context, namespace, imageRepo, verb str
 		},
 	}
 	response, err := client.LocalSubjectAccessReviews(namespace).Create(&sar)
-
 	if err != nil {
-		context.GetLogger(ctx).Errorf("OpenShift client error: %s", err)
 		if kerrors.IsUnauthorized(err) || kerrors.IsForbidden(err) {
-			return ErrOpenShiftAccessDenied
+			return &errAccessDenied{err: err}
```

Comment:
the point of having a typed error is that we can compare against it. we also don't want to surface internal errors to the end user